ECMA2Yaml/IntellisenseFileGen/Program.cs

```diff
@@ -70,68 +72,71 @@ public static void Start()
                 return;
             }
             store.Build();
-            //store.ItemsByDocId.ToDictionary(p => p.Key, p.Value.CommentId);
             var typeList = LoadTypes(store.ItemsByDocId);
             ParallelOptions opt = new ParallelOptions() { MaxDegreeOfParallelism = Environment.ProcessorCount };
             var frameworks = store.GetFrameworkIndex();
```

Comment:
once we have all the frameworks, we can use the given moniker parameter to filter out a framework list that needs to run. If this list is empty, we should warn user that an invalid moniker is given.